pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md

```diff
@@ -0,0 +1,179 @@
+---
+pagename: Conversation Tester
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+permalink: third-party-bots-conversation-tester.html
+indicator: both
+---
+
+#### Conversation tester.
+## Introduction
+By using the Conversation tester, you will be able to test your bot and identified if your bot's response have the right configuration. 
+The Conversation Tester feature aims to help you to identified possible missconfiguration in your bot. This provides three important components. First the chat window in which you can start testing your bot. Second the Logs column in which you will see if you bot response was process or had an error. And finally the JSON EDITOR column that will help you to identified the error cause.
+
+## Processed Response
+When you are testing you bot and you get a successful response, it means that your bot's response has the correct configuration. See the image below.
+<img style="width:800px" src="img/botconnectordashboard/processed_response.png">
+
+## Error Response
+When testing your bot and you get an error in the response, you will be able to see what was the JSON object that produced the error in your bot response. By hovering on the last alert you will see a tooltip that will give you the possible solutions for your error. See the image below.
+
+
+<img style="width:800px" src="img/botconnectordashboard/error_response.png">
+
+### Errors that can be found per vendor
+
+### Dialog Flow V2
+
+## Invalid Action Body
+## Description
+This error happens when the bot has an invalid body and the Google Dialog Flow V2 is unable to parse it. It can be a wrong parameter, a value or missing parameter. 
+
+## Invalid Text Body
+## Description
+This error happens when the bot's response has an invalid value in its text message and the vendor is unable to parse the content.
+
+## Invalid Structured Content
+## Description
+This error happens when the bot's configuration has an invalid content body in its structured content response and the vendor is unable to parse it.It can be wrong parameter or a value;
+
+## Invalid Text With Encoded Metadata Body
+## Description
+This error happens when the bot's configuration has an invaldid text message with encoded metadata. It can be that the encodedMetadata property is missing or the is something wrong in the encoded metadata body.
+
+## Invalid Delay Body
+## Description
+This error happens when the bot's response has an invalid delay value and the vendor is unable to parse it. It can be that a wrong delay value. 
+
+## Invalid Private Text Body
+## Description
+This error happens when the bot's response has an invalid value in its private text message and the vendor is unable to parse the content. It can be a missing property or a wrong parameter or value.
+
+## Invalid AI Vendor Response
+## Description
+This error happens when there was not possible to get any response from the vendor.
+
+#### Errors Solution
```

Comment:
Probably it is better to provide Error and Solution for each of the errors to that relevant document link in TPB. Likes for example in Invalid Delay Body for Watson refer the link to the Delay section of the Watson place. You don't have to make a separate heading named solution. You can simply after describing the error lead the user to the relevant documentation